### PR TITLE
Update resource examples

### PR DESCRIPTION
**Pull request by IanMadd**
### Description

Update the resources Hugo templates:
- Removes trailing white spaces 
- replaces the `examples_list` property with `examples`

Replace the `examples_list` with `examples`:
- In each yaml file the `examples` property is now a concatenation of the previous `examples_list`

### Check List

- [ ] Spell Check
- [X] Local build
- [X] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/resources/apt_package/_index.md [hunk 1]
```diff
@@ -140,17 +140,10 @@ properties_list:
   required: false
   description_list:
   - markdown: The version of a package to be installed or upgraded.
-examples_list:
-- example_heading: Install a package using package manager
-  text_blocks:
-  - code_block: |
```
Comment (kagarmoe) [inline content/resources/apt_package/_index.md:146]: Sorry that I'm coming late to this discussion. Did the literal block scalar just not work? I'm not wild about the hand curation needed in the solution.

Comment (IanMadd) [reply content/resources/apt_package/_index.md:146]: The issue is that all the examples need to be in one long MD string. Currently there are resource examples in `chef/chef`, but they're one long string and there's no easy way to get them from `chef/chef` to the current Hugo template. When Tim updates the resources using his rake task, then the MD will look readable not like the jumbled mess that it is here.

Comment (tas50) [reply content/resources/apt_package/_index.md:146]: Automate the world!

**Comment by tas50** [comment]
Looks good to me. I can start automating all these now. Thanks @IanMadd

**Review by tas50 — APPROVED** [review]

**Pull request merged**